katello/hooks/pre/30-upgrade.rb

```diff
@@ -37,6 +37,48 @@ def migrate_foreman
   Kafo::Helpers.execute('foreman-rake db:migrate')
 end
 
+def postgresql_10_upgrade
+  return unless facts[:os][:release][:major] == '7'
+
+  #Ensure there is at least 1x /var/lib/pgsql free disk space available on the main filesystem
+  current_postgres_dir = '/var/lib/pgsql'
+  new_postgres_dir = '/var/opt/rh/rh-postgresql10/lib/pgsql'
+
+  begin
+    until File.directory?(new_postgres_dir)
+      new_postgres_dir = File.dirname(new_postgres_dir)
+    end
+
+    postgres_size = `du -k -s #{current_postgres_dir}`.split[0].to_i
```

Comment:
available space is in bytes and this in KB.

Reply:
Editing in two places for testing, I caught htis on my live system just not in git!

Reply:
I often have the same issue :( Technically it's nowadays possible to run straight from git but I never really do this.